crates/burn-import/onnx-tests/build.rs

```diff
@@ -107,6 +107,7 @@ fn main() {
         .input("tests/sum/sum_int.onnx")
         .input("tests/tanh/tanh.onnx")
         .input("tests/tile/tile.onnx")
+        .input("tests/top_k/top_k.onnx")
```

Comment:
Looks the CI caught something I missed! This file doesn't exist anymore with your changes :)